crates/sui-e2e-tests/tests/full_node_tests.rs

```diff
@@ -1369,3 +1370,106 @@ async fn transfer_coin(
     let resp = context.execute_transaction_must_succeed(txn).await;
     Ok((object_to_send.0, sender, receiver, resp.digest, gas_object))
 }
+
+#[sim_test]
+async fn test_full_node_run_with_range_checkpoint() -> Result<(), anyhow::Error> {
```

Comment:
this test appears to work
`cargo-simtest simtest --test full_node_tests test_full_node_run_with_range_checkpoint`